CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Fixed
 
+- `DetailPage`: Fixed TS error of interface usage ([@qubis741](https://github.com/qubis741) in [#2151](https://github.com/teamleadercrm/ui/pull/2151))
```

Comment:
I wouldn't create a changelog item for this since we're not adding changelog items for converting to TS in the first place